docs/pom.xml

```diff
@@ -31,6 +31,12 @@
 
     <name>Camel Quarkus :: Documentation</name>
 
+    <properties>
+        <!-- Disable xref checks as long as there is no 3.14.x branch of Camel docs -->
```

Comment:
An issue for re-enabling and pasting it's URL here would be nice.

Reply:
@ppalaga  I didn't understand your comment